tests/scan/test_basic.py

```diff
@@ -408,6 +408,30 @@ def f_pow2(x_tm1):
         rng = np.random.default_rng(utt.fetch_seed())
         my_f(rng.uniform(size=(3,)), 4, np.int64([2, 2, 3]))
 
+    def test_no_step(self):
+        """We expect an empty output when `n_steps==0`"""
+
+        def f_pow(x_tm1):
+            return 2 * x_tm1
+
+        x_init = scalar("x")
+        values, _ = scan(f_pow, outputs_info=(x_init,), n_steps=0)
+
+        update_fn = function((x_init,), values)
+        assert update_fn(1.0).size == 0
```

Comment:
Is the returned  value 1? Why not assert the expected output directly?

Reply:
The returned values is an empty array (same behavior as JAX's for stacked outputs).

Reply:
> The returned values is an empty array (same behavior as JAX's for stacked outputs).

There's no need to reference JAX; these changes only need to fit our established operational assumptions (e.g. the `Type`s associated with each quantity).